src/main/java/com/o19s/es/explore/PostingsExplorerQuery.java

```diff
@@ -78,6 +79,10 @@ public int hashCode() {
     @Override
     public Weight createWeight(IndexSearcher searcher, ScoreMode scoreMode, float boost)
             throws IOException {
+        if (!LTRSettings.isLTRPluginEnabled()) {
```

Comment:
Possible edit is making this a "checkAndThrow" type of call as to not have the exception throw call be duplicated.

Reply:
Good point.